src/components/AssetHeader/AssetHeader.tsx

```diff
@@ -173,22 +173,23 @@ export const AssetHeader: React.FC<AssetHeaderProps> = ({ assetId, accountId })
           </Box>
         </Box>
       </Card.Body>
-      {view === View.Balance ? (
+      <Box style={{ display: view === View.Balance ? 'block' : 'none' }}>
```

Comment:
@edwardluu can you please just add a 1 line comment above explaining what this is doing and i'll gladly approve